package.json

```diff
@@ -4,32 +4,24 @@
   "description": "Sequelize adapter for Node-Casbin",
   "main": "lib/adapter.js",
   "typings": "lib/adapter.d.ts",
-  "scripts": {
-    "precommit": "lint-staged",
-    "prepublish": "yarn run lint && yarn run test && yarn build",
-    "build": "rimraf lib && tsc",
-    "lint": "tslint \"src/**/*.ts\"",
-    "fix": "tslint \"src/**/*.ts\" --fix",
-    "test": "jest"
-  },
+  "scripts": {},
```

Comment:
Why are these deleted?

Reply:
Because when I run the command `yarn install`, it run `jest test` automatically.

Reply:
I don't understand, if we don't specify ``"test": "jest"``, how do Yarn know which test the project is using? Why not Mocha, why must be jest?

Reply:
I don't understand, too. But I think the prepublish script run automatically.
```shell
[~] yarn install
yarn install v1.9.4
[1/4] 🔍  Resolving packages...
success Already up-to-date.
$ yarn run lint && yarn run test && yarn build
yarn run v1.9.4
$ tslint "src/**/*.ts"
✨  Done in 2.02s.
yarn run v1.9.4
$ jest
 FAIL  test/adapter.test.ts
  ● Test suite failed to run

    TypeError: sequelize_1.Sequelize.define is not a function

      16 | import { Sequelize } from 'sequelize';
      17 |
    > 18 | const CasbinRule = Sequelize.define('casbin_rule', {
         |                              ^
      19 |     ptype: Sequelize.STRING,
      20 |     v0: Sequelize.STRING,
      21 |     v1: Sequelize.STRING,

      at Object.<anonymous> (src/adapter.ts:18:30)
      at Object.<anonymous> (test/adapter.test.ts:16:1)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        6.664s
Ran all test suites.
error Command failed with exit code 1.
```

Reply:
If you are not sure about a thing, then simply do not change it.